pyramid/scripts/pserve.py

```diff
@@ -334,6 +341,17 @@ def serve():
                     msg = ''
                 self.out('Exiting%s (-v to see traceback)' % msg)
 
+        if self.options.browser:
+            def open_browser():
+                context = loadcontext(SERVER, app_spec, name=app_name, relative_to=base,
+                        global_conf=vars)
+                url = 'http://{host}:{port}/'.format(**context.config())
```

Comment:
Oh I don't know maybe? Host seems to always be 127.0.0.1 or 0.0.0.0 and those both work. 

What problem are you worried about?